packages/flutter/lib/src/material/drawer_header.dart

```diff
@@ -51,20 +91,93 @@ class DrawerHeader extends StatelessWidget {
           )
         )
       ),
-      child: new Stack(
-        children: <Widget>[
-          background ?? new Container(),
-          new Positioned(
-            top: statusBarHeight + 16.0,
-            left: 16.0,
-            right: 16.0,
-            bottom: 8.0,
-            child: new DefaultTextStyle(
-              style: Theme.of(context).textTheme.body2,
-              child: content
+      child: new Container(
+        padding: const EdgeInsets.only(left: 16.0, right: 16.0, bottom: 8.0),
+        decoration: config.decoration ?? new BoxDecoration(),
+        child: new Column(
+          children: <Widget>[
+            new Flexible(
+              child: new Stack(
+                children: <Widget>[
+                  new Positioned(
+                    top: statusBarHeight + 16.0,
+                    right: 0.0,
+                    child: new Row(
+                      children: config.otherAccountsPictures.map(
+                        (p) => new Container(
+                          margin: const EdgeInsets.only(left: 16.0),
+                          constraints: new BoxConstraints.tight(
+                              new Size(42.0, 42.0)),
+                          child: new ClipOval(
+                            child: p
+                          )
+                        )
+                      ).take(3).toList()
+                    )
+                  ),
+                  new Positioned(
+                    top: statusBarHeight + 16.0,
+                    child: new Container(
+                      constraints: new BoxConstraints.tight(
+                          new Size(72.0, 72.0)),
+                      child: new ClipOval(
+                        child: config.currentAccountPicture
+                      )
+                    )
+                  )
+                ]
+              )
+            ),
+            new Container(
+              height: 56.0,
+              child: new Center(
+                child: new InkWell(
+                  onTap: () {
+                    setState(() {
+                      isOpen = !isOpen;
+                    });
+
+                    if (config.onDetailsPressed != null) {
+                      config.onDetailsPressed();
+                    }
+                  },
+                  child: new Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisSize: MainAxisSize.min,
```

Comment:
Is this needed?  I would think you'd just want to specify mainAxisAlignment: MainAxisAlignment.start